linker/shared/src/main/scala/org/scalajs/linker/analyzer/Infos.scala

```diff
@@ -99,6 +100,7 @@ object Infos {
       val usedInstanceTests: List[ClassName],
       val accessedClassData: List[ClassName],
       val referencedClasses: List[ClassName],
+      val staticallyReferencedClasses: List[ClassName],
```

Comment:
I'm not sure whether this, or the additional `privateJSFieldsRead` / `privateJSFieldsWritten` is better.